learns-app-content/week-09/objectives.md

```diff
@@ -1,23 +1,29 @@
 <!-- h1, h2 already used by CTD Learns -->
 ### Topics Covered
 
-- topic 1
-- topic 2
-- topic n
+- Advanced state and useReducer
+- useContext
+- API-based sort/search
 
 ### Lesson Objectives
 
-#### Objective 1: Topic 1
+By the end of this lesson, we will:
 
-<!-- - {{blooms action verb}} sub-topic takeaway -->
-<!-- - {{blooms action verb}} sub-topic takeaway -->
+#### Objective 1: Advanced State and useReducer
 
-#### Objective 2: Topic 2
+- Discuss problems of state management in growing applications
+- Introduce the reducer pattern
+- Describe reducers, dispatches, and actions
+- Describe useReducer and how it implements the reducer pattern
+- Implement useReducer to combine related state
 
-<!-- - {{blooms action verb}} sub-topic takeaway -->
-<!-- - {{blooms action verb}} sub-topic takeaway -->
+#### Objective 2: useContext
 
-#### Objective 3: Topic 3
+- Explain the problems that that can be introduced by "prop drilling" and lifting state too high
+- Describe how useContext can be implemented in deeply nested components to avoid prop drilling
```

Comment:
Not sure if it warrants its own objective, but should we add something about explaining the pros and cons of using a context and which contexts (pun intended) are best suited for it? We could just include this in the info and not as an objective though.

Reply:
Added

- Explain the pros and cons of using useContext
- Outline scenarios best suited for context